imports/components/player_list.jsx

```diff
@@ -0,0 +1,44 @@
+export class PlayerList extends React.Component {
+  render() {
+    var players = this.props.players.map(function(player) {
+      return (<PlayerListEntry key={player._id} {...player} />);
+    });
+    var logIn = null;
+    if (!this.props.loggedIn) {
+      logIn = <li>
+        <strong>Log In</strong> to join this list
+      </li>
```

Comment:
Actually, I'm going to see if I can get eslint working off the config from my other projects, since it seems non-trivial